aws/aws-sdk-php-symfony/1.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Aws\\Symfony\\AwsBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "AWS_CREDENTIAL_KEY": "not-a-real-key",
```

Comment:
Could we use just `AWS_KEY` and `AWS_SECRET` or are those names recommended by AWS? Shorter is better IMHO.

Reply:
Done :)